### PR TITLE
Update moabs to 1.3.9.2

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`moabs`](https://bioconda.github.io/recipes/moabs/README.html): **1.3.9.0** &rarr; **1.3.9.2**

[install with bioconda](http://bioconda.github.io/recipes/moabs/README.html) [Conda](https://anaconda.org/bioconda/moabs/files)

Info | Link
-----|-----
Recipe | [`recipes/moabs`](https://github.com//bioconda/bioconda-recipes/tree/bump/moabs/recipes/moabs) (click to view/edit other files)
Releases |[https://github.com/sunnyisgalaxy/moabs/tags](https://github.com/sunnyisgalaxy/moabs/tags)
Author | `@sunnyisgalaxy`
***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Comment by lijinbio** [comment]
This PR failed in `test-macos`, but the error is unknown. Can you help suggest what is the error in this build? so that I can fix the check. Thank you.

**Comment by dpryan79** [comment]
@lijinbio The errors in the CI are:

```
03:52:37 BIOCONDA INFO (ERR) ./version:1:1: error: expected unqualified-id
03:52:37 BIOCONDA INFO (ERR) 1.5 (08 Jan 2003)
03:52:37 BIOCONDA INFO (ERR) ^
03:52:37 BIOCONDA INFO (ERR) In file included from gzstream.C:29:
03:52:37 BIOCONDA INFO (ERR) In file included from ./gzstream.h:33:
03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/iostream:37:
03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/ios:215:
03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/__locale:14:
03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/string:504:
03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/string_view:175:
03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/__string:57:
03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/algorithm:643:
03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/memory:666:
03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/atomic:553:
03:52:37 BIOCONDA INFO (ERR) ./version:1:1: error: expected unqualified-id
03:52:37 BIOCONDA INFO (ERR) 1.5 (08 Jan 2003)
03:52:37 BIOCONDA INFO (ERR) ^
03:52:37 BIOCONDA INFO (ERR) In file included from gzstream.C:29:
03:52:37 BIOCONDA INFO (ERR) In file included from ./gzstream.h:33:
03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/iostream:37:
03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/ios:215:
03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/__locale:14:
03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/string:504:
03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/string_view:175:
03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/__string:57:
03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/algorithm:643:
03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/memory:666:
03:52:37 BIOCONDA INFO (ERR) /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/atomic:1019:73: error: unknown type name 'ptrdiff_t'
03:52:37 BIOCONDA INFO (ERR) _Tp* __cxx_atomic_fetch_add(__cxx_atomic_base_impl<_Tp*> volatile* __a, ptrdiff_t __delta, memory_order __order) _NOEXCEPT {
03:52:37 BIOCONDA INFO (ERR)                                                                         ^
03:52:37 BIOCONDA INFO (ERR) /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/atomic:1024:65: error: unknown type name 'ptrdiff_t'
03:52:37 BIOCONDA INFO (ERR) _Tp* __cxx_atomic_fetch_add(__cxx_atomic_base_impl<_Tp*> * __a, ptrdiff_t __delta, memory_order __order) _NOEXCEPT {
03:52:37 BIOCONDA INFO (ERR)                                                                 ^
03:52:37 BIOCONDA INFO (ERR) /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/atomic:1040:73: error: unknown type name 'ptrdiff_t'
03:52:37 BIOCONDA INFO (ERR) _Tp* __cxx_atomic_fetch_sub(__cxx_atomic_base_impl<_Tp*> volatile* __a, ptrdiff_t __delta, memory_order __order) _NOEXCEPT {
03:52:37 BIOCONDA INFO (ERR)                                                                         ^
03:52:37 BIOCONDA INFO (ERR) /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/atomic:1045:65: error: unknown type name 'ptrdiff_t'
03:52:37 BIOCONDA INFO (ERR) _Tp* __cxx_atomic_fetch_sub(__cxx_atomic_base_impl<_Tp*> * __a, ptrdiff_t __delta, memory_order __order) _NOEXCEPT {
03:52:37 BIOCONDA INFO (ERR)                                                                 ^
03:52:37 BIOCONDA INFO (ERR) /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/atomic:1679:20: error: unknown type name 'ptrdiff_t'
03:52:37 BIOCONDA INFO (ERR)     _Tp* fetch_add(ptrdiff_t __op, memory_order __m = memory_order_seq_cst)
03:52:37 BIOCONDA INFO (ERR)                    ^
03:52:37 BIOCONDA INFO (ERR) fatal error: too many errors emitted, stopping now [-ferror-limit=]
```

**Comment by lijinbio** [comment]
> @lijinbio The errors in the CI are:
> 
> ```
> 03:52:37 BIOCONDA INFO (ERR) ./version:1:1: error: expected unqualified-id
> 03:52:37 BIOCONDA INFO (ERR) 1.5 (08 Jan 2003)
> 03:52:37 BIOCONDA INFO (ERR) ^
> 03:52:37 BIOCONDA INFO (ERR) In file included from gzstream.C:29:
> 03:52:37 BIOCONDA INFO (ERR) In file included from ./gzstream.h:33:
> 03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/iostream:37:
> 03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/ios:215:
> 03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/__locale:14:
> 03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/string:504:
> 03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/string_view:175:
> 03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/__string:57:
> 03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/algorithm:643:
> 03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/memory:666:
> 03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/atomic:553:
> 03:52:37 BIOCONDA INFO (ERR) ./version:1:1: error: expected unqualified-id
> 03:52:37 BIOCONDA INFO (ERR) 1.5 (08 Jan 2003)
> 03:52:37 BIOCONDA INFO (ERR) ^
> 03:52:37 BIOCONDA INFO (ERR) In file included from gzstream.C:29:
> 03:52:37 BIOCONDA INFO (ERR) In file included from ./gzstream.h:33:
> 03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/iostream:37:
> 03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/ios:215:
> 03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/__locale:14:
> 03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/string:504:
> 03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/string_view:175:
> 03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/__string:57:
> 03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/algorithm:643:
> 03:52:37 BIOCONDA INFO (ERR) In file included from /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/memory:666:
> 03:52:37 BIOCONDA INFO (ERR) /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/atomic:1019:73: error: unknown type name 'ptrdiff_t'
> 03:52:37 BIOCONDA INFO (ERR) _Tp* __cxx_atomic_fetch_add(__cxx_atomic_base_impl<_Tp*> volatile* __a, ptrdiff_t __delta, memory_order __order) _NOEXCEPT {
> 03:52:37 BIOCONDA INFO (ERR)                                                                         ^
> 03:52:37 BIOCONDA INFO (ERR) /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/atomic:1024:65: error: unknown type name 'ptrdiff_t'
> 03:52:37 BIOCONDA INFO (ERR) _Tp* __cxx_atomic_fetch_add(__cxx_atomic_base_impl<_Tp*> * __a, ptrdiff_t __delta, memory_order __order) _NOEXCEPT {
> 03:52:37 BIOCONDA INFO (ERR)                                                                 ^
> 03:52:37 BIOCONDA INFO (ERR) /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/atomic:1040:73: error: unknown type name 'ptrdiff_t'
> 03:52:37 BIOCONDA INFO (ERR) _Tp* __cxx_atomic_fetch_sub(__cxx_atomic_base_impl<_Tp*> volatile* __a, ptrdiff_t __delta, memory_order __order) _NOEXCEPT {
> 03:52:37 BIOCONDA INFO (ERR)                                                                         ^
> 03:52:37 BIOCONDA INFO (ERR) /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/atomic:1045:65: error: unknown type name 'ptrdiff_t'
> 03:52:37 BIOCONDA INFO (ERR) _Tp* __cxx_atomic_fetch_sub(__cxx_atomic_base_impl<_Tp*> * __a, ptrdiff_t __delta, memory_order __order) _NOEXCEPT {
> 03:52:37 BIOCONDA INFO (ERR)                                                                 ^
> 03:52:37 BIOCONDA INFO (ERR) /Users/distiller/project/miniconda/conda-bld/moabs_1589798881366/_build_env/bin/../include/c++/v1/atomic:1679:20: error: unknown type name 'ptrdiff_t'
> 03:52:37 BIOCONDA INFO (ERR)     _Tp* fetch_add(ptrdiff_t __op, memory_order __m = memory_order_seq_cst)
> 03:52:37 BIOCONDA INFO (ERR)                    ^
> 03:52:37 BIOCONDA INFO (ERR) fatal error: too many errors emitted, stopping now [-ferror-limit=]
> ```

@dpryan79 Thank you for the error message. The first error is about `version`. This file is a text file, and should not be used in compiling. I.e.,

```
$ cat gzstream/version 
1.5 (08 Jan 2003)
```
May I ask how this file causes the error in the compiling? Should removal of this file help?

The second error is about including `<iostream>`.

```
$ sed -n '29 p' gzstream/gzstream.C
#include <gzstream.h>
$ sed -n '33 p' gzstream/gzstream.h
#include <iostream>
```
Can you help suggest why the source code can not include `<iostream>`?

Basically, these library files (gzstream/*) are not changed since many releases before, and these two errors occur for the first time. Can you help suggest how to resolve this CI build error? Thank you.

**Comment by dpryan79** [comment]
I have no idea how your package works, I will be of little help.

**Comment by sfmetadb** [comment]
@dpryan79 Hi Devon, I have updated my package, and the CI check failed even in `test-linux`. In fact, the build is successful, but the last step failed due to some unknown error. 

```
01:24:27 BIOCONDA INFO BUILD SUCCESS moabs-1.3.9.2-pl526r40h7cd8e1f_0.tar.bz2
01:24:27 BIOCONDA INFO TEST START via mulled-build recipes/moabs
01:24:29 BIOCONDA INFO (COMMAND) mulled-build build-and-test moabs=1.3.9.2--pl526r40h7cd8e1f_0 -n biocontainers --test moabs --help --channels conda-forge,file:///home/circleci/project/miniconda/conda-bld,bioconda,defaults --involucro-path /home/circleci/project/miniconda/bin/involucro
01:24:30 BIOCONDA INFO (OUT) /home/circleci/project/miniconda/bin/involucro -v=2 -f /home/circleci/project/miniconda/lib/python3.7/site-packages/galaxy/tools/deps/mulled/invfile.lua -set CHANNELS='conda-forge,file:///home/circleci/project/miniconda/conda-bld,bioconda,defaults' -set TARGETS='moabs=1.3.9.2=pl526r40h7cd8e1f_0' -set REPO='quay.io/biocontainers/moabs:1.3.9.2--pl526r40h7cd8e1f_0' -set BINDS='build/dist:/usr/local/,/home/circleci/project/miniconda/conda-bld:/home/circleci/project/miniconda/conda-bld' -set CONDA_IMAGE='quay.io/dpryan79/mulled_container:latest' -set TEST='moabs --help' build-and-test
01:24:30 BIOCONDA INFO (ERR) Passing program arguments as a string may be a security hazard if combined with untrusted input
01:24:30 BIOCONDA INFO (ERR) [May 25 01:24:30] DEBU Run file [/home/circleci/project/miniconda/lib/python3.7/site-packages/galaxy/tools/deps/mulled/invfile.lua]
01:24:30 BIOCONDA INFO (ERR) [May 25 01:24:30] INFO Invoke Task [build]
01:24:30 BIOCONDA INFO (ERR) [May 25 01:24:30] STEP Run image [quay.io/dpryan79/mulled_container:latest] with command [[rm -rf /data/dist]]
01:24:30 BIOCONDA INFO (ERR) [May 25 01:24:30] DEBU Creating container [step-0ee8c8c035]
01:24:30 BIOCONDA INFO (ERR) [May 25 01:24:30] DEBU Image [quay.io/dpryan79/mulled_container:latest] not present, pulling it
01:24:30 BIOCONDA INFO (ERR) [May 25 01:24:30] DEBU Pull Image [quay.io/dpryan79/mulled_container:latest]
01:24:30 BIOCONDA INFO (ERR) [May 25 01:24:30] ERRO Task processing failed: API error (500): {"message":"Get https://quay.io/v2/dpryan79/mulled_container/manifests/latest: unauthorized: Invalid Username or Password"}
01:24:30 BIOCONDA INFO (ERR) 
.01:25:30 BIOCONDA INFO (OUT) .
01:25:30 BIOCONDA ERROR COMMAND FAILED (exited with 1): mulled-build build-and-test moabs=1.3.9.2--pl526r40h7cd8e1f_0 -n biocontainers --test moabs --help --channels conda-forge,file:///home/circleci/project/miniconda/conda-bld,bioconda,defaults --involucro-path /home/circleci/project/miniconda/bin/involucro

01:25:30 BIOCONDA ERROR TEST FAILED: recipes/moabs
01:25:30 BIOCONDA INFO (COMMAND) conda build purge
01:25:31 BIOCONDA ERROR BUILD SUMMARY: of 1 recipes, 1 failed and 0 were skipped. Details of recipes and environments follow.
01:25:31 BIOCONDA ERROR BUILD SUMMARY: FAILED recipe recipes/moabs

Exited with code exit status 1
```

Can you help suggest what is this error? and how to fix it? Thank you.

Best regards,
Jin
